### PR TITLE
Create Summarize Discussion.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

**Comment by sarikasu** [comment]
reached out to authors 3/2 for exact instructions

File: tasks/Summarize Discussion.mds [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Roles 
+> No roles were assigned to participants however participants were assigned to four person same-sex groups for the tasks 
+
+
+# Criteria
+## Performance calculation
+> The quality of the written reports were assessed on four dimensions developed by Hackman, Jones and McGrath (1967) (https://psycnet.apa.org/record/1967-10399-001): 
+> The dimensions are: length, originality, quality of presentation, adequacy and creativity and one dimension assessing the length (Quantity)
+> All group products were rated on scales assessing five dimensions using a 7- pile sort-resort procedure by three judges- two assessed each dimension and all three
```
Comment (katelynbottcher) [inline tasks/Summarize Discussion.mds:45]: I found a more descriptive explanation of the "sort-resort" procedure here https://psycnet-apa-org.proxy.library.upenn.edu/fulltext/1967-10399-001.pdf if you wanted to go into more detail but I'm not sure that it's necessary

Comment (sarikasu) [reply tasks/Summarize Discussion.mds:45]: I will add this to references, great find!

**Review by katelynbottcher — APPROVED** [review]

**Comment by katelynbottcher** [comment]
Hi! Sorry for missing this, but this also needs to be changed to task.md it's currently task.mds.

**Comment by sarikasu** [comment]
shoot sorry about that! Is it best to create a new pull request and have this deleted or how can I change it now?

**Comment by markwhiting** [comment]
Hey, I actually fixed this in commit https://github.com/Watts-Lab/task-mapping/commit/ba8874c41af0c927deb704045cf8fe6c1e5f6743 when I was cleaning some other stuff. Usually however, you'd do this by:

1. going to master
2. finding the file there
3. making the adjustment
4. when you are done it would ask you if you want to make a pull request with that change. You would, and then it could be merged.

If this was about more than 1 file it would be easier to do the branch based approach (i.e., starting by making a new branch and then editing that), instead of editing on master. These are actually the same thing under the hood (i.e., both are branching, committing, and PRing), but GitHub makes it a little more convent for a single file edit in this way.

**Comment by sarikasu** [comment]
Ok thank you! I will keep a closer eye on this moving forward!

**Pull request merged**